pytissueoptics/scene/intersection/intersectionFinder.py

```diff
@@ -13,10 +14,19 @@
 
 
 @dataclass
-class Intersection:
+class IntersectionInfo:
```

Comment:
I don't understand the need for this class. Why not save the polygon ref in the Intersection?

Reply:
Htere must be something i'm not seeing...

Reply:
Felt better segregated this way, but it adds noise and the cost of unused ref is not really considerable so I will put everything in a single Intersection dataclass instead.